### PR TITLE
Add test for keeper `mntr` command

### PR DESCRIPTION
**Pull request by alesapin**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/integration/test_keeper_mntr_pressure/test.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    node1_thread = threading.Thread(target=lambda: go_mntr(node1.name))
+    node2_thread = threading.Thread(target=lambda: go_mntr(node2.name))
+    node3_thread = threading.Thread(target=lambda: go_mntr(node3.name))
+    node1_thread.start()
+    node2_thread.start()
+    node3_thread.start()
+
+    node2.stop_clickhouse()
+    node3.stop_clickhouse()
+    node1.stop_clickhouse()
```
Comment (antonio2368) [inline tests/integration/test_keeper_mntr_pressure/test.py:78]: maybe wait until the node1 yields leadership?

Comment (antonio2368) [reply tests/integration/test_keeper_mntr_pressure/test.py:78]: https://github.com/ClickHouse/ClickHouse/blob/master/tests/integration/test_keeper_force_recovery/test.py#L166
You can do something like this

File: src/Coordination/KeeperDispatcher.cpp [hunk 2]
```diff
@@ -652,10 +652,11 @@ uint64_t KeeperDispatcher::getSnapDirSize() const
 Keeper4LWInfo KeeperDispatcher::getKeeper4LWInfo() const
 {
     Keeper4LWInfo result;
-    result.is_follower = server->isFollower();
+    result.is_follower = isFollower();
```
Comment (antonio2368) [inline src/Coordination/KeeperDispatcher.cpp:655]: We need to store the value of `isLeader` in a variable and not call the function twice.

Comment (alesapin) [reply src/Coordination/KeeperDispatcher.cpp:655]: Agree

**Review by antonio2368 — APPROVED** [review]

**Comment by alesapin** [comment]
Database replicated tests broken for some reason, let's try to fix.

**Pull request merged**